docs/guide/core-widgets.md

```diff
@@ -189,7 +233,12 @@ module.exports = {
       ],
       styles: [
         // Your own default styles
-      ]
+      ],
+      pickerOptions: {
+        // Your own default picker options
```

Comment:
Say "color picker options, if you use the color picker for text" really clearly here, this option has a general name and could be confusing